src/components/common/ShareAppButton.tsx

```diff
@@ -1,12 +1,21 @@
 import React, { ElementType } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { Alert, Share, StyleSheet, View } from 'react-native';
 import { Icon, TouchableOpacity } from '.';
-import { toggleChangeLanguage } from '../../actions/LocaleActions';
-
 
 const ShareAppButton: ElementType = () => {
+  const onShare = async () => {
+    try {
+      await Share.share({
+        message:
+                    'Hey, Please download. https://govextra.gov.il/ministry-of-health/corona/corona-virus',
```

Comment:
Maybe it's better to also support localization here based on the selected language

Reply:
the link the is already globalized. regarding the "Hey ,Please donwload" text this should be part of the i18n effort opened on other issue

Reply:
PS - you're watching an outdated commit

Reply:
Yes, I realized that. And still, regarding the "Hey, Please download" do we have some global ticket for i18n issues?

Reply:
there are several. French (feat), Russian(fix) and Spanish. 
this is a feature branch, i strongly suggest we move all i18n discussions to another issue.

Reply:
But this is jus features or fixes to add or fix some Grammarly typos in the language, it's not related to the fact that your string should be added to the English i18n, it's part of the feature

Reply:
I mean, add the text to the English, Hebrew section here: https://github.com/MohGovIL/hamagen-react-native/blob/master/src/locale/texts.json

It's better than having hard-coded text

Reply:
there are several issues with us changing the text here.

1. the text probably will be rephrased in each language differently, so we can use google translate, but it's preferred that the language maintainers would make sure it makes sense.
2. unless we know the recipient language, I think sticking with English is a reasonable idea.

Reply:
I can add the texts, its no brainer, can someone from the team can confirm the phrasing? or supply translations?

Reply:
1. You're right, that's why I said it's better to at least have English and Hebrew and for the rest, we should tag other maintainers for help.

2. Probably the recipient language is the same language as the one that shares the app.
But...It's just my opinion.

Anyway, good job. this feature is necessary, especially for elders.

Reply:
thanks man, appreciated 🙏